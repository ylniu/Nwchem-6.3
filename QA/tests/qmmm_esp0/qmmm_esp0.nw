Title "QM/MM of Lys monomer"


memory total 800 Mb


start lys

echo


basis "ao basis" 
  O library "6-31G*"
  H library "6-31G*"
  C library "6-31G*"
  N library "6-31G*"
  F library "6-31G*"
end


# There is a net charge on NH3 group
charge 1.0

# Specify any SCF details
dft
 iterations 100
end

md
 system lys_qmmm
 cutoff 1.9   qmmm 1.8
end

task qmmm dft energy
task qmmm esp

