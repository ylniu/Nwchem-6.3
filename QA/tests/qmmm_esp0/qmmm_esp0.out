HP-MPI licensed for ISV application.
All connections between all procs tested: SUCCESS
 argument  1 = qmmm_esp0.nw



============================== echo of input deck ==============================
Title "QM/MM of Lys monomer"


memory total 800 Mb


start lys

echo


basis "ao basis"
  O library "6-31G*"
  H library "6-31G*"
  C library "6-31G*"
  N library "6-31G*"
  F library "6-31G*"
end


# There is a net charge on NH3 group
charge 1.0

# Specify any SCF details
dft
 iterations 100
end

md
 system lys_qmmm
 cutoff 1.9   qmmm 1.8
end

task qmmm dft energy
task qmmm esp

================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = cu0login1
    program       = /hptc_cluster/scicons/apps/nwchem-6.0.oct19/bin/LINUX64/nwchem
    date          = Mon Nov  1 15:20:50 2010

    compiled      = Thu_Oct_28_07:10:53_2010
    source        = /home/scicons/user/kurt/nwchem-6.0-release-pgf90-final/
    nwchem branch = 6.0
    input         = qmmm_esp0.nw
    prefix        = lys.
    data base     = ./lys.db
    status        = startup
    nproc         =        1
    time left     =     -1s



           Memory information
           ------------------

    heap     =   26214401 doubles =    200.0 Mbytes
    stack    =   26214401 doubles =    200.0 Mbytes
    global   =   52428800 doubles =    400.0 Mbytes (distinct from heap & stack)
    total    =  104857602 doubles =    800.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                               QM/MM of Lys monomer
                               --------------------
                      Basis "ao basis" -> "" (cartesian)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.48467170E+03  0.001831
  1 S  8.25234950E+02  0.013950
  1 S  1.88046960E+02  0.068445
  1 S  5.29645000E+01  0.232714
  1 S  1.68975700E+01  0.470193
  1 S  5.79963530E+00  0.358521

  2 S  1.55396160E+01 -0.110778
  2 S  3.59993360E+00 -0.148026
  2 S  1.01376180E+00  1.130767

  3 P  1.55396160E+01  0.070874
  3 P  3.59993360E+00  0.339753
  3 P  1.01376180E+00  0.727159

  4 S  2.70005800E-01  1.000000

  5 P  2.70005800E-01  1.000000

  6 D  8.00000000E-01  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757

  2 S  1.61277800E-01  1.000000

  C (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.04752490E+03  0.001835
  1 S  4.57369510E+02  0.014037
  1 S  1.03948690E+02  0.068843
  1 S  2.92101550E+01  0.232184
  1 S  9.28666300E+00  0.467941
  1 S  3.16392700E+00  0.362312

  2 S  7.86827240E+00 -0.119332
  2 S  1.88128850E+00 -0.160854
  2 S  5.44249300E-01  1.143456

  3 P  7.86827240E+00  0.068999
  3 P  1.88128850E+00  0.316424
  3 P  5.44249300E-01  0.744308

  4 S  1.68714400E-01  1.000000

  5 P  1.68714400E-01  1.000000

  6 D  8.00000000E-01  1.000000

  N (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  4.17351100E+03  0.001835
  1 S  6.27457900E+02  0.013995
  1 S  1.42902100E+02  0.068587
  1 S  4.02343300E+01  0.232241
  1 S  1.28202100E+01  0.469070
  1 S  4.39043700E+00  0.360455

  2 S  1.16263580E+01 -0.114961
  2 S  2.71628000E+00 -0.169118
  2 S  7.72218000E-01  1.145852

  3 P  1.16263580E+01  0.067580
  3 P  2.71628000E+00  0.323907
  3 P  7.72218000E-01  0.740895

  4 S  2.12031300E-01  1.000000

  5 P  2.12031300E-01  1.000000

  6 D  8.00000000E-01  1.000000

  F (Fluorine)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  7.00171309E+03  0.001820
  1 S  1.05136609E+03  0.013916
  1 S  2.39285690E+02  0.068405
  1 S  6.73974453E+01  0.233186
  1 S  2.15199573E+01  0.471267
  1 S  7.40310130E+00  0.356619

  2 S  2.08479528E+01 -0.108507
  2 S  4.80830834E+00 -0.146452
  2 S  1.34406986E+00  1.128689

  3 P  2.08479528E+01  0.071629
  3 P  4.80830834E+00  0.345912
  3 P  1.34406986E+00  0.722470

  4 S  3.58151393E-01  1.000000

  5 P  3.58151393E-01  1.000000

  6 D  8.00000000E-01  1.000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31G*                  6       15   3s2p1d
 H                           6-31G*                  2        2   2s
 C                           6-31G*                  6       15   3s2p1d
 N                           6-31G*                  6       15   3s2p1d
 F                           6-31G*                  6       15   3s2p1d


 ****************************************************
 *      NWChem QM/MM Interface Module               *
 *                                                  *
 *      Developed Marat Valiev 2004                 *
 ****************************************************
                    QM/MM Interface Parameters
                    --------------------------
 operation             task:operation        energy                                                        
 reference energy      qmmm:uqmatm               0.000000  <--- Warning zero value is not advisable !!!    
 bqzone radius         qmmm:bqzone              18.000000                                                  
 excluded bq's         qmmm:bq_exclude       none                                                          
 link atom type        qmmm:linkatm          hydrogens                                                     
 optimization method   qmmm:optimization     bfgs                                                          
 ecp on link atoms     qmmm:link_ecp         auto                                                          
 interface api         qmmm:interface        qm                                                            
 ------------------------------------------------------


 number of quantum atoms                        4
 Total number of Bq charges                       20
 Number of solute Bq charges                       20
 Number of solvent Bq charges                        0
 Total Bq charge:     1.361800000000000     
 Total number of active Bq charges                        20
 frozen_density,esp  F  F

                                 NWChem DFT Module
                                 -----------------


                               QM/MM of Lys monomer


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     5
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     1
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations: 100
          AO basis - number of functions:    23
                     number of shells:    14
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                        Slater Exchange Functional  1.000 local    
                      VWN V Correlation Functional  1.000 local    

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          N                   0.65       49           6.0       434
          H                   0.35       45           7.0       434
          H_L                 0.35       45           7.0       434
          Grid pruning is: on 
          Number of quadrature shells:   229
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters        100 iters           100 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08

 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     

      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -56.28666840
 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     

 Renormalizing density from      11.00 to     10

      Non-variational initial energy
      ------------------------------

 Total energy =     -52.760408
 1-e energy   =     -96.819820
 2-e energy   =      27.485383
 HOMO         =      -1.226353
 LUMO         =      -0.447751

 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     
   Time after variat. SCF:      0.2
   Time prior to 1st pass:      0.2

 #quartets = 5.565D+03 #integrals = 3.621D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./lys.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      3        Max. records in file   = ******
 No. of bits per label  =      8        No. of bits per value  =     64

 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     

 Grid_pts file          = ./lys.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =     37        Max. recs in file   =  44834395


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):        0.00                   2
          Stack Space remaining (MW):       26.21            26207392

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1    -56.0228140544 -7.26D+01  6.74D-02  2.65D+00     0.7
 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     
 d= 0,ls=0.0,diis     2    -55.5808184013  4.42D-01  5.16D-02  5.37D+00     0.9
 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     
 d= 0,ls=0.0,diis     3    -56.2294244893 -6.49D-01  5.99D-03  1.12D-01     1.1
 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     
 d= 0,ls=0.0,diis     4    -56.2416456468 -1.22D-02  5.86D-04  5.06D-04     1.3
 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     
 d= 0,ls=0.0,diis     5    -56.2417066105 -6.10D-05  1.89D-05  3.29D-07     1.5
 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     
 d= 0,ls=0.0,diis     6    -56.2417066487 -3.82D-08  3.53D-06  9.37D-09     1.6
 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     


         Total DFT energy =      -56.241706648689
      One electron energy =     -103.936494934256
           Coulomb energy =       38.590966073439
    Exchange-Corr. energy =       -7.470206671266
 Nuclear repulsion energy =       16.574028883394

 Numeric. integr. density =       10.000000502582

     Total iterative time =      1.5s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-1.439895D+01
              MO Center= -1.5D-01,  3.2D+00,  1.7D+00, r^2= 2.0D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.989983  1 N  s          

 Vector    2  Occ=2.000000D+00  E=-1.257543D+00
              MO Center= -1.3D-01,  3.1D+00,  1.7D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.494849  1 N  s                  2      0.429234  1 N  s          
     1     -0.208463  1 N  s          

 Vector    3  Occ=2.000000D+00  E=-8.965143D-01
              MO Center= -1.0D-01,  3.1D+00,  1.6D+00, r^2= 9.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.386322  1 N  pz                 4      0.285469  1 N  py         
    22     -0.270072  5 H  s                  9      0.218606  1 N  pz         
     3     -0.185688  1 N  px                 8      0.157604  1 N  py         

 Vector    4  Occ=2.000000D+00  E=-8.700419D-01
              MO Center= -3.2D-01,  3.1D+00,  1.9D+00, r^2= 9.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5     -0.332190  1 N  pz                 3     -0.308457  1 N  px         
     4      0.249411  1 N  py                 9     -0.216792  1 N  pz         
    16     -0.214482  2 H  s                 20      0.209815  4 H  s          
     7     -0.201209  1 N  px                 8      0.162595  1 N  py         

 Vector    5  Occ=2.000000D+00  E=-8.690151D-01
              MO Center= -6.7D-02,  3.4D+00,  1.8D+00, r^2= 9.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3     -0.371366  1 N  px                 4     -0.352677  1 N  py         
    18     -0.246963  3 H  s                  7     -0.244636  1 N  px         
     8     -0.231936  1 N  py         

 Vector    6  Occ=0.000000D+00  E=-4.103102D-01
              MO Center= -1.3D-01,  3.1D+00,  1.6D+00, r^2= 3.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6     -1.448048  1 N  s                 23      0.850738  5 H  s          
    17      0.561855  2 H  s                 21      0.553575  4 H  s          
    19      0.488419  3 H  s                  2     -0.312762  1 N  s          

 Vector    7  Occ=0.000000D+00  E=-3.394099D-01
              MO Center= -3.1D-01,  2.8D+00,  2.3D+00, r^2= 2.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      1.153505  2 H  s                  9     -0.701964  1 N  pz         
    21     -0.660547  4 H  s                 23     -0.542037  5 H  s          
     5     -0.422472  1 N  pz                16      0.269888  2 H  s          
     7     -0.246538  1 N  px                 8      0.202367  1 N  py         
     3     -0.163279  1 N  px                20     -0.160901  4 H  s          

 Vector    8  Occ=0.000000D+00  E=-3.388993D-01
              MO Center= -4.1D-01,  3.0D+00,  1.2D+00, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    23      1.294058  5 H  s                 21     -0.937252  4 H  s          
     7     -0.562965  1 N  px                 8      0.467373  1 N  py         
     9      0.410961  1 N  pz                 3     -0.337344  1 N  px         
     4      0.249366  1 N  py                20     -0.213910  4 H  s          
     5      0.210404  1 N  pz                22      0.171469  5 H  s          

 Vector    9  Occ=0.000000D+00  E=-3.348262D-01
              MO Center=  2.5D-01,  3.9D+00,  1.8D+00, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    19     -1.234002  3 H  s                  8      0.604542  1 N  py         
     7      0.455818  1 N  px                21      0.430597  4 H  s          
     4      0.372501  1 N  py                17      0.368461  2 H  s          
    23      0.361093  5 H  s                 18     -0.297874  3 H  s          
     3      0.295891  1 N  px         

 Vector   10  Occ=0.000000D+00  E= 1.777373D-01
              MO Center= -2.0D-01,  3.3D+00,  1.8D+00, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.920389  1 N  pz                22      0.778194  5 H  s          
     8      0.681150  1 N  py                 7     -0.435350  1 N  px         
     5     -0.374307  1 N  pz                 4     -0.272431  1 N  py         
    14      0.255780  1 N  dyz               16     -0.228209  2 H  s          
    20     -0.219455  4 H  s                  6     -0.197164  1 N  s          

 Vector   11  Occ=0.000000D+00  E= 2.000476D-01
              MO Center= -1.7D-01,  3.1D+00,  1.7D+00, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7     -1.039331  1 N  px                18      0.613106  3 H  s          
     8     -0.610371  1 N  py                 3      0.538565  1 N  px         
    20     -0.494393  4 H  s                  4      0.316618  1 N  py         
    11      0.271184  1 N  dxy        

 Vector   12  Occ=0.000000D+00  E= 2.004967D-01
              MO Center= -1.1D-01,  3.2D+00,  1.7D+00, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.798554  1 N  py                 9     -0.784416  1 N  pz         
    16      0.644451  2 H  s                 20     -0.434655  4 H  s          
     7     -0.420984  1 N  px                 4     -0.412354  1 N  py         
     5      0.405442  1 N  pz                 3      0.216520  1 N  px         
    18     -0.210016  3 H  s                 14     -0.170007  1 N  dyz        

 Vector   13  Occ=0.000000D+00  E= 3.255070D-01
              MO Center= -4.2D-01,  3.2D+00,  2.1D+00, r^2= 2.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6     -1.142979  1 N  s                 17      0.985543  2 H  s          
    21      0.893603  4 H  s                 16     -0.678260  2 H  s          
    20     -0.640657  4 H  s                  2      0.586465  1 N  s          
     9     -0.562934  1 N  pz                19      0.495927  3 H  s          
    18     -0.483303  3 H  s                  7      0.379044  1 N  px         

 Vector   14  Occ=0.000000D+00  E= 3.337376D-01
              MO Center= -6.3D-01,  3.1D+00,  2.1D+00, r^2= 2.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    21     -1.334509  4 H  s                 17      1.309701  2 H  s          
     9     -0.805399  1 N  pz                 7     -0.763533  1 N  px         
     8      0.630648  1 N  py                20      0.569674  4 H  s          
    16     -0.536902  2 H  s                  5      0.491223  1 N  pz         
     3      0.464080  1 N  px                 4     -0.382844  1 N  py         

 Vector   15  Occ=0.000000D+00  E= 3.382008D-01
              MO Center=  2.4D-01,  3.9D+00,  1.8D+00, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    19     -1.663565  3 H  s                  8      0.897861  1 N  py         
     7      0.863387  1 N  px                18      0.751501  3 H  s          
    21      0.638740  4 H  s                 17      0.546081  2 H  s          
     4     -0.541368  1 N  py                 3     -0.525298  1 N  px         
    20     -0.218570  4 H  s                  6      0.196220  1 N  s          

 Vector   16  Occ=0.000000D+00  E= 3.546925D-01
              MO Center=  1.3D-01,  2.7D+00,  1.1D+00, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    23      1.720430  5 H  s                  9      0.891160  1 N  pz         
     6     -0.850750  1 N  s                 22     -0.833116  5 H  s          
     8      0.675094  1 N  py                 5     -0.592305  1 N  pz         
     4     -0.450243  1 N  py                 7     -0.425041  1 N  px         
     2      0.416824  1 N  s                  3      0.281194  1 N  px         

 Vector   17  Occ=0.000000D+00  E= 7.526684D-01
              MO Center= -1.6D-01,  3.2D+00,  1.8D+00, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      4.136446  1 N  s                  2     -1.665866  1 N  s          
    23     -0.668349  5 H  s                 19     -0.527880  3 H  s          
    21     -0.519851  4 H  s                 17     -0.518787  2 H  s          
    16     -0.423402  2 H  s                 20     -0.423337  4 H  s          
    18     -0.421949  3 H  s                 22     -0.401443  5 H  s          

 Nuclear repulsion energy =    14.93765463616649     
 Bq nuclear interaction energy =    1.636374247227050     

 center of mass
 --------------
 x =  -0.04262067 y =   5.57709859 z =   2.75288345

 moments of inertia (a.u.)
 ------------------
          35.842319343893           5.112885380087           6.501916279220
           5.112885380087          29.949217967806         -11.246511479617
           6.501916279220         -11.246511479617          24.495563939115

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      1.000000     -5.000000     -5.000000     11.000000

     1   1 0 0     -0.359830      1.459268      1.459268     -3.278365
     1   0 1 0      6.158248    -30.025264    -30.025264     66.208776
     1   0 0 1      3.472522    -16.494380    -16.494380     36.461282

     2   2 0 0     -2.824602     -5.002078     -5.002078      7.179554
     2   1 1 0     -2.070491      8.749723      8.749723    -19.569937
     2   1 0 1     -1.080595      4.790250      4.790250    -10.661095
     2   0 2 0     34.830286   -184.849020   -184.849020    404.528325
     2   0 1 1     21.096458    -99.003154    -99.003154    219.102767
     2   0 0 2      8.800593    -58.950714    -58.950714    126.702020


 Parallel integral file used       1 records with       0 large values

 Bq nuclear interaction energy =    1.636374247227050     
------------------------------------------------------------------------
 QM/MM Energy
------------------------------------------------------------------------
 quantum energy                       -56.241706649 (-.147663E+06  kjoule/mol)
 quantum energy adjusted              -56.241706649 (-.147663E+06  kjoule/mol)
 quantum energy internal              -56.380629793 (-.148027E+06  kjoule/mol)
 Bq-nuclear energy                      1.636374247 (0.429630E+04  kjoule/mol)
 Bq-electron energy                    -1.497451103 (-.393156E+04  kjoule/mol)
 classical energy                       0.068152561 (0.178935E+03  kjoule/mol)
 total qmmm energy                    -56.173554087 (-.147484E+06  kjoule/mol)
------------------------------------------------------------------------


 Task  times  cpu:        1.6s     wall:        2.5s


                                NWChem Input Module
                                -------------------


 ****************************************************
 *      NWChem QM/MM Interface Module               *
 *                                                  *
 *      Developed Marat Valiev 2004                 *
 ****************************************************
                    QM/MM Interface Parameters
                    --------------------------
 operation             task:operation        energy                                                        
 reference energy      qmmm:uqmatm               0.000000  <--- Warning zero value is not advisable !!!    
 bqzone radius         qmmm:bqzone              18.000000                                                  
 excluded bq's         qmmm:bq_exclude       none                                                          
 link atom type        qmmm:linkatm          hydrogens                                                     
 optimization method   qmmm:optimization     bfgs                                                          
 ecp on link atoms     qmmm:link_ecp         auto                                                          
 interface api         qmmm:interface        qm                                                            
 ------------------------------------------------------


 number of quantum atoms                        4
 Total number of Bq charges                       20
 Number of solute Bq charges                       20
 Number of solvent Bq charges                        0
 Total Bq charge:     1.361800000000000     


                     NWChem Electrostatic Potential Fit Module
                     -----------------------------------------


 Atom parameters

 Number of atoms is                                    5
 Number of basis functions is                         23


 Grid parameters

 Maximum number of grid points is                   1632
 Number of grid points is                           1631
 Grid range                                     0.300000 nm
 Grid spacing                                   0.050000 nm
 Probe radius                                   0.070000 nm
 Atom radius factor                             1.000000

 Atomic radii

    1    0.100000
    7    0.140000

 Recovering from shell      0

11/01/10   15:20:53  er shell      1 of     14
11/01/10   15:20:53  er shell      2 of     14
11/01/10   15:20:53  er shell      3 of     14
11/01/10   15:20:53  er shell      4 of     14
11/01/10   15:20:53  er shell      5 of     14
11/01/10   15:20:53  er shell      6 of     14
11/01/10   15:20:53  er shell      7 of     14
11/01/10   15:20:53  er shell      8 of     14
11/01/10   15:20:53  er shell      9 of     14
11/01/10   15:20:53  er shell     10 of     14
11/01/10   15:20:53  er shell     11 of     14
11/01/10   15:20:53  er shell     12 of     14
11/01/10   15:20:53  er shell     13 of     14
11/01/10   15:20:53  er shell     14 of     14

 Recovery file deleted



    Atom        Coordinates                           Charge

                                      ESP   
                                            
 
    1 N   -0.015   0.318   0.175   -0.714410
    2 H   -0.010   0.261   0.274    0.442334
    3 H    0.049   0.413   0.181    0.450677
    4 H   -0.125   0.348   0.154    0.444251
    5 H_   0.020   0.257   0.097    0.377148
                                ------------
                                    1.000000

 Dipole moment                      0.132139

 Quadrupole moment Qxx              0.443788
                   Qyy              0.018399
                   Qzz             -0.462187

 RMS deviation kJ/mol               0.119560

 RMS deviation %                    0.422562

 Task  times  cpu:        0.5s     wall:        0.7s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  136      136     1.11e+04  822     7383        0        0        0     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             3.18e+06 6.05e+05 1.83e+06 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 188720 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        59	       132
	current total bytes		         0	         0
	maximum total bytes		 214933088	 393752216
	maximum total K-bytes		    214934	    393753
	maximum total M-bytes		       215	       394


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        2.1s     wall:        3.7s
